src/js/media-controller.js

```diff
@@ -143,14 +144,31 @@ class MediaController extends MediaContainer {
 
     } else if (
       attrName === Attributes.DEFAULT_SUBTITLES &&
-      newValue !== oldValue &&
-      newValue === ''
+      newValue !== oldValue
     ) {
-      toggleSubsCaps(this, true);
+      this._mediaStore.dispatch({
+        type: 'configchangerequest',
```

Comment:
I'm continuing to like "option" over "config", i.e. `optionchangerequest`.

Reply:
awesome I'm down